intervention_compassion/models/compassion_intervention.py

```diff
@@ -849,6 +850,20 @@ def json_to_data(self, json, mapping_name=None):
             json["ICP"] = json["ICP"].split("; ")
         return super().json_to_data(json, mapping_name)
 
+    @api.model
+    def send_notif_before_expiration(self):
+        threshold_date = fields.datetime.today() + relativedelta(days=1)
```

Comment:
1 day only is risky, if it's the weekend or that person is not working that day... it should be at least 7 days.